src/asm/rgbasm.5

```diff
@@ -875,43 +875,45 @@ and at signs
 However, they must begin with either a letter or an underscore.
 Periods
 .Sq \&.
-are allowed exclusively in labels, as described below.
+are allowed exclusively in label names, as described below.
 A symbol cannot have the same name as a reserved keyword.
-.Ss Label declaration
+.Ss Label definition
 One of the assembler's main tasks is to keep track of addresses for you, so you can work with meaningful names instead of "magic" numbers.
+Labels enable just that.
+A label definition is made of a label name at the beginning of a line,
+.Sy immediately
+followed by one or two colons.
 .Pp
-This can be done in a number of ways:
+Any label whose name does not contain a period is a global label.
+Defining a global label sets it as the current label scope, until the next global one.
+.Pp
+Any label whose name contains a period is a local label; more than one period, however, is not allowed.
```

Comment:
For someone new to labels, this whole section might be confusing. How about subheadings for "global label" (`Label:` or `Label::`), "local label" (`Label.local`, colons optional, containing a single period), and "scope-local label" (`.label`, colons optional, scoped to the previous global).

Reply:
Mandoc is only made to sustain two-level headings; a third level has to be artifical (e.g. `Sy`), but then you can't link to it. I guess that we could e.g. use a list with such "artificial" headings (`Bl -tag` IIRC), if it's small enough.

Regardless, I think we should separate global labels from local ones, and introduce the special `.local` syntax there, not make a third heading for it, since it's only syntax sugar on top of a local label decl.